instrumentation/spring/spring-webmvc-3.1/spring-webmvc-3.1-auto/src/test/resources/logback.xml

```diff
@@ -14,5 +14,6 @@
   </root>
 
   <logger name="io.opentelemetry.auto" level="debug"/>
+  <logger name="io.opentelemetry.javaagent" level="debug"/>
```

Comment:
Remove that line above. I believe it is obsolete

Reply:
Done.